Assets/Scripts/Player/Cinemachine_Shake.cs

```diff
@@ -0,0 +1,51 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using Cinemachine;
+public class Cinemachine_Shake : MonoBehaviour
+{
+    public static Cinemachine_Shake Instance { get; private set; }
```

Comment:
very good, you've used a static reference!
I'm very glad you went with this approach